### DISCUSSION TITLE
How many point、path、vehicle can put into model?

### DISCUSSION THREAD

**Discussion by mbj2014** [openTCS/opentcs#168] [Q&A]
My System is intel i5-1240p,64G,SSD，win11
when I generated a model with my program, have 100 rows, 100 cols grid net map, 200 vehicles, only load modle into ModelEditor user more then 10min!

somebody test the limit?

**Comment by mbj2014** [comment]
and when loading, cpu 15%，mem use 20G（include my system and other program）,disk io 0%. I don't know ModelEditor doing something but is very slowly!

**Reply by swltr** [reply]
This is a known issue: The Model Editor and the Operations Desk applications do indeed need an unacceptable amount of time when loading models with a relatively large number of elements. The larger the model, the more time they need. The reason for this seems to be in the JHotDraw library. Switching to a UI based on more modern technology is definitely desirable.

Note that the kernel is not affected - it works fluently with large models, too.

**Reply by mbj2014** [reply]
Is there a recommended UI solution project based on modern technology to replace the low-performance official model editor and Operations Desk?

**Reply by swltr** [reply]
At the moment, alternative UIs are not available. Code contributions or funding to create one would be highly welcome.
